test/flycheck-test.el

```diff
@@ -3881,6 +3881,33 @@ Why not:
     (flycheck-ert-should-syntax-check
      "language/rust/lib-main/src/main.rs" 'rust-mode)))
 
+(flycheck-ert-def-checker-test rust-cargo rust notes
+  ;; We only get notes on the first build. Ensure we start from a
+  ;; clean directory each time.
+  (call-process-shell-command
```

Comment:
I initially resisted `call-process` because of all the `nil` parameters. We don't need a shell, so I'll change.